setup.py

```diff
@@ -3,16 +3,17 @@
 # Setup script for PyPI; use CMakeFile.txt to build the example application
 
 from setuptools import setup
+from pybind11 import __version__
```

Comment:
@wjakob this can work for now because pybind has no dependency that are imported from the base `__init__.py` and would not be installed yet at this point. The earlier method discussed in https://github.com/SylvainCorlay/pybind11/commit/cf19b85e689edd1ba94b71d9ce24921c1af0d415